src/doc/rustc/src/codegen-options/index.md

```diff
@@ -516,6 +516,50 @@ enabled. It takes one of the following values:
 * `y`, `yes`, `on`, `true` or no value: enable rpath.
 * `n`, `no`, `off` or `false`: disable rpath (the default).
 
+## sanitize
+
+Sanitizers are a set of compiler-based runtime error detection tools that
+instrument programs to detect bugs during execution. They work by instrumenting
+code at compile time and runtime to monitor program behavior and detect specific
+classes of errors at runtime. Sanitizers enable precise, low-overhead runtime
+bug detection, improving software reliability and security.
+
+This option allows for use of one or more of these sanitizers:
+
+  * [AddressSanitizer
+    (ASan)](https://doc.rust-lang.org/rustc/sanitizers.html#addresssanitizer):
```

Comment:
All links to books should be relative (this helps with offline viewing and linkchecking).

```suggestion
    (ASan)](../sanitizers.md#addresssanitizer):
```